packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -32,35 +33,53 @@ import CardHeader from '../../../components/CardHeader';
 
 export const viewName = 'complete-reset-password';
 
-export type CompleteResetPasswordProps = {
-  email: string;
-  forceAuth?: boolean;
-  linkStatus: LinkStatus;
-  // resetPasswordConfirm will be obtained from the relier
-  // resetPasswordConfirm determines if sync warning is shown
-  resetPasswordConfirm?: boolean;
-};
-
 type FormData = {
   newPassword: string;
   confirmPassword: string;
 };
 
 type LinkStatus = 'expired' | 'damaged' | 'valid';
 
-const CompleteResetPassword = ({
-  email,
-  forceAuth = false,
-  linkStatus,
-  resetPasswordConfirm,
-}: CompleteResetPasswordProps & RouteComponentProps) => {
-  usePageViewEvent(viewName, REACT_ENTRYPOINT);
+const CompleteResetPassword = (_: RouteComponentProps) => {
+  logPageViewEvent(viewName, REACT_ENTRYPOINT);
 
   const [passwordMatchErrorText, setPasswordMatchErrorText] =
     useState<string>('');
-  const alertBar = useAlertBar();
   const navigate = useNavigate();
-  const ftlMsgResolver = useFtlMsgResolver();
+  const account = useAccount();
+  const [linkStatus, setLinkStatus] = useState<LinkStatus>('valid');
```

Comment:
Just thinking outloud, I think I'm going to make a hook (and the statuses set to enums) we can use for this for when we need it in more than one place like in the ticket I've been working on and use it here too. I'll see if I can get that `email!` type further down a bit cleaner too.